src/components/GamesTill.vue

```diff
@@ -1,35 +1,34 @@
-<template>
-  <md-card class="md-layout-item md-size-30 md-small-size-100">
-    <md-card-header>
-      <div class="md-title" id="page-title">To Pass {{player.name}} ({{player.total}})</div>
-      <div class="md-subhead">{{player.total - lebron}} points</div>
-    </md-card-header>
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+export default class Gamestill extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      error: null,
+      loading: true
+    }
+  }
+  static propTypes = {
+    average: PropTypes.number,
```

Comment:
Guessing some react guide led you to use PropTypes. Shouldn't be necessary in react with typescript as you can just use `number` or `string` or `object`